pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/ProviderUrlAssert.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import java.net.URL;
+
+public class ProviderUrlAssert {
```

Comment:
Do you plan to have more functionalities in this Assert class? If not maybe we can flatten it to one class and call `.testInteraction()` with or without URL depending on if it's set?